dfget/config/config.go

```diff
@@ -240,35 +243,45 @@ func NewConfig() *Config {
 	cfg.Sign = fmt.Sprintf("%d-%.3f",
 		os.Getpid(), float64(time.Now().UnixNano())/float64(time.Second))
 
-	if currentUser, err := user.Current(); err == nil {
-		cfg.User = currentUser.Username
-		cfg.WorkHome = path.Join(currentUser.HomeDir, ".small-dragonfly")
-		cfg.RV.MetaPath = path.Join(cfg.WorkHome, "meta", "host.meta")
-		cfg.RV.SystemDataDir = path.Join(cfg.WorkHome, "data")
-		cfg.RV.FileLength = -1
-	} else {
-		panic(fmt.Errorf("get user error: %s", err))
+	currentUser, err := user.Current()
+	if err != nil {
+		util.Printer.Println(fmt.Sprintf("get user error: %s", err))
+		os.Exit(CodeGetUserError)
```

Comment:
I don't suggest to use `os.Exit` in inner packages. It cannot recover any more and not easy to test.

Reply:
It is unreasonable to rely heavily on the user's home directory. We should make it possible to customize by parameters. WDYT? If that, I will finish it in the later pull request and  record a `TODO` task temporarily.